hrjobcontract/js/src/job-contract/services/utils.js

```diff
@@ -127,6 +144,33 @@ define([
 
                 return deffered.promise;
             },
+
+            /**
+             * Returns a promise that resolves the an int with the number of Public Holidays in the
+             * current Absence Period
+             *
+             * @returns {Promise}
+             */
+            getNumberOfPublicHolidaysInCurrentPeriod: function() {
+              var deffered = $q.defer();
+
+              API.resource('PublicHoliday', 'getcountforcurrentperiod', {
+                sequential: 1
+              }).get(function (data) {
+                var number = parseInt(data.result);
+
+                if(isNaN(number)) {
+                  number = 0;
+                }
```

Comment:
Just realized you can rewrite this whole bit as such:
```js
var number = parseInt(data.result) || 0;

deffered.resolve(number);
```

Or, if you like one-liners:
```js
deferred.resolve(parseInt(data.result) || 0);
```